site/content/integrate/apps/quickstart/quick-start-go/_index.md

```diff
@@ -1,288 +1,99 @@
 ---
-title: "Quick start guide (Go)"
-heading: "Write a Mattermost app in Go"
-description: "This quick start guide will walk you through the basics of writing a Mattermost App in Go."
+title: "Hello World in Go"
+heading: "Create a Hello World Mattermost app in Go"
+description: "This quick start guide will walk you through the basics of creating a hello world Mattermost app in Go."
 weight: 10
 aliases:
   - /integrate/apps/quick-start-go/
 ---
 
-This quick start guide explains the basics of writing a Mattermost app. In this guide you will build an app that:
+This quick start guide will walk you through the basics of creating a hello world Mattermost app in Go. In this guide you will review an app that:
 
-- Contains a `manifest.json`, declares itself an HTTP application that acts as a bot, and attaches to locations in the user interface.
+- Contains a `manifest.json`, declares itself an HTTP application that acts as a bot, and attaches to locations in the Mattermost interface.
 - Attaches the form `send-modal` in its `bindings` to a button in the channel header, and the form `send` to a `/helloworld` command.
 - Contains a `send` function that sends a parameterized message back to the user.
 - Contains a `send-modal` function that forces displaying the `send` form as a modal.
 
 ## Prerequisites
 
-Before you can start with your app, you first need to set up a local developer environment following the [server]({{< ref "/contribute/server/developer-setup" >}}) and [webapp]({{< ref "/contribute/webapp/developer-setup" >}}) setup guides. You must enable the apps feature flag before starting the Mattermost server by setting the environment variable `MM_FEATUREFLAGS_AppsEnabled` to `true` by e.g. adding `export MM_FEATUREFLAGS_AppsEnabled=true` to your `.bashrc` or using `make run-server MM_FEATUREFLAGS_AppsEnabled=true`.
+Before you can start with your app, you first need to set up your environment by following the [developer setup guide]({{< ref "/integrate/apps/quickstart" >}}).
 
-In the System Console, ensure that the following are set to **true**:
+You also need at least `go1.16` installed. Please follow the [official guide](https://golang.org/doc/install) to install the latest version.
 
-- `Enable Bot Account Creation`
-- `Enable OAuth 2.0 Service Provider`
+## Download and start the app
 
-You also need at least `go1.16` installed. Please follow the guide [here](https://golang.org/doc/install) to install the latest version.
+In the same [mattermost-app-examples repository](https://github.com/mattermost/mattermost-app-examples) you cloned via the [developer setup guide]({{< ref "/integrate/apps/quickstart" >}}) above, navigate to the [`golang/hello-world`](https://github.com/mattermost/mattermost-app-examples/tree/master/golang/hello-world) directory and start the Docker compose:
```

Comment:
I think [stack is a loaded word](https://nickjanetakis.com/blog/docker-tip-23-docker-compose-vs-docker-stack#:~:text=Docker%20Compose%20is%20an%20official,Docker%20containers%20through%20Docker%20Swarm.) in the world of Docker, so I'd like to avoid using it so people don't think `swarm` is involved at all.